tox.ini

```diff
@@ -1,12 +1,21 @@
 [tox]
-envlist = tests, flake8
+envlist = tests, marionette, flake8
 
 [testenv]
 passenv = DISPLAY PYTEST_ADDOPTS
 deps =
+    mock
+    pytest==3.0.4
+    pytest-selenium
+    pytest-xdist==1.15.0
+commands = py.test {posargs} 
+
+[testenv:marionette]
+passenv = DISPLAY PYTEST_ADDOPTS
+deps =
     marionette_driver==2.1.0
     mock
-    pytest~=3.0.0
+    pytest==3.0.4
     pytest-selenium
     pytest-xdist==1.15.0
 commands = py.test {posargs}
```

Comment:
I would suggest either adding the path to the tests here to isolate marionette:

```ini
commands = pytest {posargs:tests/marionette}
```

Then in `[testenv]` include an `--ignore` argument in the `commands` to exclude them.